x/gamm/keeper/pool_service_test.go

```diff
@@ -430,6 +430,86 @@ func (suite *KeeperTestSuite) TestExitPool() {
 	}
 }
 
+// TestJoinPoolExitPool_InverseRelationship tests that joining pool and exiting pool
+// guarantees same amount in and out
+func (suite *KeeperTestSuite) TestJoinPoolExitPool_InverseRelationship() {
```

Comment:
Let's name it TestJoinPoolExitPool_Balancer_InverseRelationship since we can have a similar test for a stable swap pool?